frontend/database/143_add_show_diff_problem.sql

```diff
@@ -0,0 +1,3 @@
+-- Alter Problems table
+
+ALTER TABLE `Problems` ADD COLUMN `show_diff` enum('none', 'examples', 'all') NOT NULL DEFAULT 'none' COMMENT 'Determina si el problema es educativo y debe mostrar diferencias en casos de ejemplos, en todos o en ninguno.' AFTER `quality_seal`;
```

Comment:
nit:

```suggestion
ALTER TABLE
  `Problems`
ADD COLUMN
  `show_diff` enum('none', 'examples', 'all') NOT NULL DEFAULT 'none' COMMENT 'Determina si el problema es educativo y debe mostrar diferencias en casos de ejemplos, en todos o en ninguno.'
AFTER `quality_seal`;
```

para consistencia con los otros archivos `.sql`.

Reply:
Listo